src/com/facebook/buck/apple/AppleBundle.java

```diff
@@ -409,7 +421,7 @@ public boolean isLegacyWatchApp() {
     }
 
     if (!frameworks.isEmpty()) {
-      Path frameworksDestinationPath = bundleRoot.resolve(this.destinations.getFrameworksPath());
```

Comment:
Please revert this line and remove `getFrameworksPath`, should not expose too much info outside of enclosed class like `AppleBundle`. Or make `getFrameworksPath` private, I would prefer removal still